packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -42,10 +42,43 @@ export default class FunctionRun extends Command {
       path: flags.path,
       userProvidedConfigName: flags.config,
       callback: async (_app, ourFunction) => {
+        let functionExport = '_start'
+
+        if (flags.export !== undefined) {
+          this.log('Using export ${flags.export} from the --export flag.')
+          functionExport = flags.export
+        } else if (
+          ourFunction.configuration.targeting !== undefined &&
```

Comment:
Is this the indentation that the CLI linting/prettier config suggested?

Reply:
`:shakes-fist-at-prettier:`